locale/moretrees.fr.tr

```diff
@@ -211,10 +211,10 @@ Willow Tree Trunk Stair=Escalier en tronc de saule
 Yellow=jaune
 
 # init.lua
-Sapling=
-Tree=
-Wooden Planks=
-Leaves=
-Wooden Fence=
-Wooden Fence Rail=
-Wooden Fence Gate=
+Sapling=Jeune Arbre
+Tree=Arbre
+Wooden Planks=Planches
+Leaves=Feuilles
+Wooden Fence=Barrière en bois
+Wooden Fence Rail=Clôture en bois
```

Comment:
Clôture is a synonym of barrière. Probably the better word for "Fence".
So, "Wooden Fence Rail" should be "Rail de clôture en bois"